src/Orb/OpenApi.hs

```diff
@@ -595,7 +595,14 @@ data SchemaInfo = SchemaInfo
   , openApiSchema :: OpenApi.Schema
   , schemaComponents :: Map.Map T.Text SchemaInfo
   }
-  deriving (Eq)
+
+isSameSchemaInfo :: SchemaInfo -> SchemaInfo -> Bool
+isSameSchemaInfo (SchemaInfo a1 b1 c1 _ d1 e1) (SchemaInfo a2 b2 c2 _ d2 e2) =
+  a1 == a2
+    && b1 == b2
+    && c1 == c2
+    && d1 == d2
```

Comment:
I would really prefer to see the names of what's actually being compared here. Otherwise it's hard to read this function and understand what choices were made about what to compare.

Reply:
Addressed in 565a67f4e934c04fbe72c04f821c596c3163166c